provision.yml

```diff
@@ -8,7 +8,7 @@
 
     - role: nginx
       vhosts:
-        - name: staging.stockholmsaltspa.se
+        - name: "{{ hostname }}"
```

Comment:
Hope it's obvious why this is needed! Read from inventory variables.

Reply:
Only thing I had problems with, everything else ran through smoothly on test.stockholmsaltspa.se provisioning.